utilities/pipelines/resourcePublish/Get-ModulesToUpdate.ps1

```diff
@@ -324,12 +365,13 @@ Generates a hashtable with template file paths to update with a new version.
 Generates a hashtable with template file paths to update with a new version.
 
 .PARAMETER TemplateFilePath
-Path to a deploy.bicep/json file.
+Mandatory. Path to a deploy.bicep/json file.
 
 .EXAMPLE
 Get-ModulesToUpdate -TemplateFilePath 'C:\Repos\Azure\ResourceModules\arm\Microsoft.Storage\storageAccounts\tableServices\tables\deploy.bicep'
 
 
+#TODO: Generates a hashtable with template file paths to update with a new version.
```

Comment:
Is this TODO supposted to go into main? Would rather suggest to use an issue to track intended future changes

Reply:
Resolved, please check now.